package.json

```diff
@@ -55,12 +55,12 @@
   },
   "version": "12.2.0-next.0",
   "dependencies": {
-    "@angular/animations": "^12.1.2",
-    "@angular/common": "^12.1.2",
-    "@angular/compiler": "^12.1.2",
-    "@angular/core": "^12.1.2",
-    "@angular/forms": "^12.1.2",
-    "@angular/platform-browser": "^12.1.2",
+    "@angular/animations": "12.1.2",
+    "@angular/common": "12.1.2",
+    "@angular/compiler": "12.1.2",
+    "@angular/core": "12.1.2",
+    "@angular/forms": "12.1.2",
```

Comment:
Why do we have to lock down to a specific version?

Reply:
If we'd not lock them there, then this lock file update would have updated to the most recent patch/minor. This would be acceptable but I figured we want to explicitly update Angular versions together with it being reflected in the `package.json`